### ISSUE TITLE
[core] Crash when launching glfw app on Linux

### ISSUE THREAD

**Issue by tmpsantos** [mapbox/mapbox-gl-native#4909]
Release mode, crashes right after the window pops up.

Reverting #4900 fixes.

```
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `./mapbox-glfw'.
Program terminated with signal SIGSEGV, Segmentation fault.
bt#0  0x000000000051d37e in mbgl::GeometryTileFeatureExtractor::getValue (this=0x7f10e92aaf70, key=...)
    at ../../src/mbgl/tile/geometry_tile.cpp:10
10	    return feature.getValue(key);
[Current thread is 1 (Thread 0x7f10e92af700 (LWP 27117))]
(gdb) bt
#0  0x000000000051d37e in mbgl::GeometryTileFeatureExtractor::getValue (this=0x7f10e92aaf70, key="scalerank")
    at ../../src/mbgl/tile/geometry_tile.cpp:10
#1  0x00000000004cbfc5 in mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>::operator() (this=this@entry=0x7f10e92aaf60, 
    filter=...) at ../../src/mbgl/style/filter_evaluator.hpp:51
#2  0x00000000004c6e28 in mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:302
#3  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., 
    v=...) at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#4  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#5  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#6  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#7  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#8  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFi---Type <return> to continue, or q <return> to quit---
lter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#9  mapbox::util::detail::dispatcher<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool, mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::apply_const (
    f=..., v=...) at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#10 mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>::visit<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool> (f=..., v=...)
    at /opt/tmpsantos/Projects/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:782
#11 mbgl::SymbolBucket::parseFeatures (this=this@entry=0x7f10d400b3f0, layer=..., filter=...)
    at ../../src/mbgl/renderer/symbol_bucket.cpp:118
#12 0x00000000004b6130 in mbgl::SymbolLayer::createBucket (this=0x18831a0, parameters=...)
    at ../../src/mbgl/layer/symbol_layer.cpp:167
#13 0x00000000005574fe in mbgl::TileWorker::parseLayer (this=this@entry=0x172db68, layer=layer@entry=0x18831a0, geometryTile=...)
    at ../../src/mbgl/tile/tile_worker.cpp:162
#14 0x00000000005582dd in mbgl::TileWorker::parseAllLayers (this=0x172db68, layers_=std::vector of length 0, capacity 0, 
    geometryTile=std::unique_ptr<const mbgl::GeometryTile> containing 0x172aa80, config=...)
    at ../../src/mbgl/tile/tile_worker.cpp:58
#15 0x000000000052e607 in mbgl::Worker::Impl::parseGeometryTile(mbgl::TileWorker*, std::vector<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> >, std::allocator<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> > > >, std::unique_ptr<mbgl::GeometryTile, std::default_delete<mbgl::GeometryTile> >, mbgl::PlacementConfig, std::function<void (mapbox::util::variant<mbgl::TileParseResultBuckets, std::__exception_ptr::exception_ptr>)>) (this=<optimized out>, 
    worker=<optimized out>, layers=..., tile=..., config=..., callback=...) at ../../src/mbgl/util/worker.cpp:37
#16 0x000000000053172f in _ZZN4mbgl4util6ThreadINS_6Worker4ImplEE4bindIMS3_FvPNS_10TileWorkerESt6vectorISt10unique_ptrINS_10StyleLayerESt14default_deleteISA_EESaISD_EES9_INS_12GeometryTileESB_ISG_EENS_15PlacementConfigESt8functionIFvN6mapbox4util7variantIJNS_22TileParseResultBucketsENSt15__exception_ptr13exception_ptrEEEEEEEEEDaT_ENKUlDpOT_E_clIJS7_SF_SI_SJ_ZNS0_7RunLoop18invokeWithCallbackIS10_RST_JS7_SF_SI_RSJ_EEES9_INS_12AsyncRequestESB_IS16_EEOSW_OT0_DpOT1_EUlSZ_E_EEEDaSZ_ (__closure=0x1885e70)
    at ../../src/mbgl/util/thread.hpp:75
#17 _ZN4mbgl4util7RunLoop7InvokerIZNS0_6ThreadINS_6Worker4ImplEE4bindIMS5_FvPNS_10TileWorkerESt6vectorISt10unique_ptrINS_10StyleLayerESt14default_deleteISC_EESaISF_EESB_INS_12GeometryTileESD_ISI_EENS_15PlacementConfigESt8functionIFvN6mapbox4util7variantIJNS_22TileParseResultBucketsENSt15__exception_ptr13exception_ptrEEEEEEEEEDaT_EUlDpOT_E_St5tupleIJS9_SH_SK_SL_ZNS1_18invokeWithCallbackIS12_---Type
```

/cc @jfirebaugh

**Comment by nuket** [comment]
Seeing the same segfault:

```
(gdb) run
Starting program: /home/max/devsrc/mapbox-gl-native/build/linux-x86_64/Release/mapbox-glfw 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

================================================================================

- Press `S` to cycle through bundled styles
- Press `X` to reset the transform
- Press `N` to reset north
- Press `R` to toggle any available `night` style class
- Press `Z` to cycle through north orientations
- Press `A` to cycle through Mapbox offices in the world + dateline monument

- Press `1` through `6` to add increasing numbers of point annotations for testing
- Press `7` through `0` to add increasing numbers of shape annotations for testing

- Press `Q` to remove annotations
- Press `P` to add a random custom runtime imagery annotation
- Press `W` to pop the last-added annotation off

- `Control` + mouse drag to rotate
- `Shift` + mouse drag to tilt

- Press `Tab` to cycle through the map debug options
- Press `Esc` to quit

================================================================================

[New Thread 0x7fffebe17700 (LWP 10317)]
[New Thread 0x7fffeb616700 (LWP 10318)]
[New Thread 0x7fffeae15700 (LWP 10319)]
[New Thread 0x7fffea614700 (LWP 10320)]
[New Thread 0x7fffe9e13700 (LWP 10321)]
[New Thread 0x7fffe9612700 (LWP 10323)]
[New Thread 0x7fffe8d11700 (LWP 10324)]
[New Thread 0x7fffd3fff700 (LWP 10325)]
[Thread 0x7fffd3fff700 (LWP 10325) exited]
[Thread 0x7fffe8d11700 (LWP 10324) exited]

Program received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fffea614700 (LWP 10320)]
0x000000000053146d in mbgl::GeometryTileFeatureExtractor::getValue (this=0x7fffea610040, key="scalerank") at ../../src/mbgl/tile/geometry_tile.cpp:10
10	    return feature.getValue(key);
(gdb) bt
#0  0x000000000053146d in mbgl::GeometryTileFeatureExtractor::getValue (this=0x7fffea610040, key="scalerank") at ../../src/mbgl/tile/geometry_tile.cpp:10
#1  0x00000000004f5965 in mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>::operator() (this=this@entry=0x7fffea610030, filter=...) at ../../src/mbgl/style/filter_evaluator.hpp:51
#2  0x00000000004f1444 in apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:302
#3  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#4  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#5  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#6  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#7  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#8  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#9  apply_const (f=..., v=...) at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:306
#10 visit<mbgl::FilterEvaluator<mbgl::GeometryTileFeatureExtractor>&, mapbox::util::variant<mbgl::NullFilter, mbgl::EqualsFilter, mbgl::NotEqualsFilter, mbgl::LessThanFilter, mbgl::LessThanEqualsFilter, mbgl::GreaterThanFilter, mbgl::GreaterThanEqualsFilter, mbgl::InFilter, mbgl::NotInFilter, mbgl::AnyFilter, mbgl::AllFilter, mbgl::NoneFilter>, bool> (f=..., v=...)
    at /home/max/devsrc/mapbox-gl-native/mason_packages/headers/variant/1.1.0/include/mapbox/variant.hpp:782
#11 mbgl::SymbolBucket::parseFeatures (this=this@entry=0x7fffd4032320, layer=..., filter=...) at ../../src/mbgl/renderer/symbol_bucket.cpp:123
#12 0x0000000000472ddf in mbgl::SymbolLayer::createBucket (this=0x14a37c0, parameters=...) at ../../src/mbgl/layer/symbol_layer.cpp:169
#13 0x0000000000542de5 in mbgl::TileWorker::parseLayer (this=this@entry=0x136a0d8, layer=layer@entry=0x14a37c0) at ../../src/mbgl/tile/tile_worker.cpp:169
#14 0x0000000000547db2 in mbgl::TileWorker::parseAllLayers (this=0x136a0d8, layers_=std::vector of length 0, capacity 0, geometryTile_=std::unique_ptr<const mbgl::GeometryTile> containing 0x0, config=...)
    at ../../src/mbgl/tile/tile_worker.cpp:60
#15 0x0000000000529497 in mbgl::Worker::Impl::parseGeometryTile(mbgl::TileWorker*, std::vector<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> >, std::allocator<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> > > >, std::unique_ptr<mbgl::GeometryTile, std::default_delete<mbgl::GeometryTile> >, mbgl::PlacementConfig, std::function<void (mapbox::util::variant<mbgl::TileParseResultData, std::__exception_ptr::exception_ptr>)>) (this=<optimized out>, worker=<optimized out>, layers=..., tile=..., config=..., callback=...) at ../../src/mbgl/util/worker.cpp:38
#16 0x000000000052862f in operator()<mbgl::TileWorker*, std::vector<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> >, std::allocator<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> > > >, std::unique_ptr<mbgl::GeometryTile, std::default_delete<mbgl::GeometryTile> >, mbgl::PlacementConfig, mbgl::util::RunLoop::invokeWithCallback(Fn&&, Cb&&, Args&& ...) [with Fn = mbgl::util::Thread<Object>::bind(Fn) [with Fn = void (mbgl::Worker::Impl::*)(mbgl::TileWorker*, std::vector<std::unique_ptr<mbgl::StyleLayer> >, std::unique_ptr<mbgl::GeometryTile>, mbgl::PlacementConfig, std::function<void(mapbox::util::variant<mbgl::TileParseResultData, std::__exception_ptr::exception_ptr>)>); Object = mbgl::Worker::Impl]::<lambda(auto:3&& ...)>; Cb = std::function<void(mapbox::util::variant<mbgl::TileParseResultData, std::__exception_ptr::exception_ptr>)>&; Args = {mbgl::TileWorker*, std::vector<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> >, std::allocator<std::unique_ptr<mbgl::StyleLayer, std::default_delete<mbgl::StyleLayer> > > >, std::unique_ptr<mbgl::GeometryTile, std::default_delete<mbgl::GeometryTile> >, mbgl::PlacementConfig&}]::<lambda(auto:1&& ...)> > (__closure=0x14a60b0)
    at ../../src/mbgl/util/thread.hpp:75
#17 invoke<0ul, 1ul, 2ul, 3ul, 4ul> (this=0x14a6070) at ../../include/mbgl/util/run_loop.hpp:148
#18 _ZN4mbgl4util7RunLoop7InvokerIZNS0_6ThreadINS_6Worker4ImplEE4bindIMS5_FvPNS_10TileWorkerESt6vectorISt10unique_ptrINS_10StyleLayerESt14default_deleteISC_EESaISF_EESB_INS_12GeometryTileESD_ISI_EENS_15PlacementConfigESt8functionIFvN6mapbox4util7variantIJNS_19TileParseResultDataENSt15__exception_ptr13exception_ptrEEEEEEEEEDaT_EUlDpOT_E_St5tupleIJS9_SH_SK_SL_ZNS1_18invokeWithCallbackIS12_RSV_JS9_SH_SK_RSL_EEESB_INS_12AsyncRequestESD_IS17_EEOSY_OT0_DpOT1_EUlS11_E_EEEclEv (this=0x14a6070) at ../../include/mbgl/util/run_loop.hpp:128
#19 0x00000000004a3465 in mbgl::util::RunLoop::process (this=0x7fffea613dc0) at ../../include/mbgl/util/run_loop.hpp:172
#20 0x00000000006e84fd in uv__async_event (loop=0x7fffd4000c60, w=<optimized out>, nevents=<optimized out>) at src/unix/async.c:92
#21 0x00000000006e8659 in uv__async_io (loop=0x7fffd4000c60, w=0x7fffd4000e28, events=<optimized out>) at src/unix/async.c:132
#22 0x00000000006f1e7e in uv__io_poll (loop=0x7fffd4000c60, timeout=-1) at src/unix/linux-core.c:345
#23 0x00000000006e8f53 in uv_run (loop=0x7fffd4000c60, mode=mode@entry=UV_RUN_DEFAULT) at src/unix/core.c:341
#24 0x00000000004a0b40 in mbgl::util::RunLoop::run (this=this@entry=0x7fffea613dc0) at ../../platform/default/run_loop.cpp:155
#25 0x0000000000526f53 in run<std::tuple<> > (params=<optimized out>, this=0xd7a9d0) at ../../src/mbgl/util/thread.hpp:126
#26 operator() (__closure=<optimized out>) at ../../src/mbgl/util/thread.hpp:110
#27 _M_invoke<> (this=<optimized out>) at /usr/include/c++/5/functional:1531
#28 operator() (this=<optimized out>) at /usr/include/c++/5/functional:1520
#29 std::thread::_Impl<std::_Bind_simple<mbgl::util::Thread<mbgl::Worker::Impl>::Thread<>(mbgl::util::ThreadContext const&)::{lambda()#1} ()> >::_M_run() (this=<optimized out>) at /usr/include/c++/5/thread:115
#30 0x00007ffff5db9390 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#31 0x00007ffff67c7182 in start_thread (arg=0x7fffea614700) at pthread_create.c:312
#32 0x00007ffff584a47d in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111
(gdb) quit
A debugging session is active.

	Inferior 1 [process 10313] will be killed.

Quit anyway? (y or n) y 
max@unu-warship:~/devsrc/mapbox-gl-native/build/linux-x86_64/Release$ uname -a
Linux unu-warship 4.2.0-35-generic #40~14.04.1-Ubuntu SMP Fri Mar 18 16:37:35 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
max@unu-warship:~/devsrc/mapbox-gl-native/build/linux-x86_64/Release$ lsb_release
No LSB modules are available.
max@unu-warship:~/devsrc/mapbox-gl-native/build/linux-x86_64/Release$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.4 LTS
Release:	14.04
Codename:	trusty

$ g++-5 -v
Using built-in specs.
COLLECT_GCC=g++-5
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/5/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 5.3.0-3ubuntu1~14.04' --with-bugurl=file:///usr/share/doc/gcc-5/README.Bugs --enable-languages=c,ada,c++,java,go,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-5 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=gcc4-compatible --disable-libstdcxx-dual-abi --enable-gnu-unique-object --disable-vtable-verify --enable-libmpx --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-5-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-5-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-5-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 5.3.0 20151204 (Ubuntu 5.3.0-3ubuntu1~14.04) 

```

**Comment by jfirebaugh** [comment]
I'm not able to reproduce this on OS X or Ubuntu 12.04.

**Comment by tmpsantos** [comment]
> I'm not able to reproduce this on OS X or Ubuntu 12.04

Have you tried debug build or Linux or running on valgrind? I get the crash on Ubuntu 16.04.

**Comment by brunoabinader** [comment]
Fix from #4935 fixes this, closing.
